File: docs-old/datatools/codesnippet/VisualBasic/how-to--delete-records-in-a-database_1.vb
        ' Locate the row to delete.
        Dim oldRegionRow As NorthwindDataSet.RegionRow
        oldRegionRow = NorthwindDataSet._Region.FindByRegionID(5)

        ' Delete the row from the dataset
        oldRegionRow.Delete()

        ' Delete the row from the database
        Me.RegionTableAdapter.Update(Me.NorthwindDataSet._Region)

File: docs/data-tools/codesnippet/CSharp/how-to-delete-records-in-a-database_1.cs
            // Locate the row to delete.
            NorthwindDataSet.RegionRow oldRegionRow;
            oldRegionRow = northwindDataSet.Region.FindByRegionID(5);

            // Delete the row from the dataset
            oldRegionRow.Delete();

            // Delete the row from the database
            this.regionTableAdapter.Update(this.northwindDataSet.Region);